ui/src/app/kvm/components/KVMHeaderForms/ComposeForm/StorageTable/PoolSelect/PoolSelect.tsx

```diff
@@ -33,7 +34,9 @@ const generateDropdownContent = (
   requests: RequestMap,
   selectPool: SelectPool
 ): JSX.Element => {
-  const poolsArray = Object.entries(pod.resources.storage_pools);
+  const poolsArray = Object.entries<KVMStoragePoolResource>(
+    pod.resources.storage_pools
+  );
```

Comment:
Turns out that by default `Object.entries` returns the type `[string, any][]` and needs a generic to be typesafe.